install_themes/action.yml

```diff
@@ -54,3 +58,54 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
+      shell: bash
+      run: |
+          echo "Deprecation Warning:"
+          echo "---------------------"
+          echo -e "\033[0;31m### This actions are deprecated and may not be supported ${{ inputs.install_theme}} \033[0m"
+          exit 1
+
+    - name: 'Output report'
+      shell: bash
+      run: |
+        for FILE in $(find reports -type f|sort); do
+          cat "${FILE}" >>github_report.md
+        done
+        cat github_report.md >>"${GITHUB_STEP_SUMMARY}"
+
+        for FILE in $(find slack_reports -type f|sort); do
+          cat "${FILE}" >>slack_report.md
+        done
+        SLACK_MARKDOWN=$(cat slack_report.md|sed 's/$/\\n/g' | tr -d '\n')
+        cat >slack_deprecated.json <<EOF
+        {
+          "blocks": [
+            {
+              "type": "header",
+              "text": {
+                "type": "plain_text",
+                "emoji": true,
+                "text": "${{inputs.title}} ${{ steps.process_slack.outputs.overall_status}}"
```

Comment:
The variables used in here don't exist. You should put an explicit title for your deprecation warning here. A good tip is to copy this JSON and paste it into https://app.slack.com/block-kit-builder/T03LW1LM5#%7B%22blocks%22:%5B%5D%7D to see, how it looks there. Or to go the other way around, use the slack builder to design your message and copy the resulting json into here